src/AppContainer.tsx

```diff
@@ -131,19 +131,19 @@ const AppContainer: FunctionComponent<Props> = ({ initialLoadComplete, documentI
         
     }
 
-    const toolBarHeight = 70
-    const H = height - toolBarHeight - 5
-    const hMargin = 30
+    const appBarHeight = 48 // hard-coded for now - must agree with theme.ts
```

Comment:
Little tweaks throughout, not done in the most systematic way. Will want to clean this up later on.